LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentAnalytics.kt

```diff
@@ -1,17 +1,14 @@
 package com.braintreepayments.api
 
-internal enum class LocalPaymentAnalytics(@JvmField val event: String) {
+internal object LocalPaymentAnalytics {
 
     // Conversion Events
-    PAYMENT_STARTED("local-payment:start-payment:started"),
-    PAYMENT_SUCCEEDED("local-payment:start-payment:succeeded"),
-    PAYMENT_FAILED("local-payment:start-payment:failed"),
-    PAYMENT_CANCELED("local-payment:start-payment:browser-login:canceled"),
+    const val PAYMENT_STARTED = "local-payment:start-payment:started"
+    const val PAYMENT_SUCCEEDED = "local-payment:start-payment:succeeded"
+    const val PAYMENT_FAILED = "local-payment:start-payment:failed"
+    const val PAYMENT_CANCELED = "local-payment:start-payment:browser-login:canceled"
 
     // Browser Presentation Events
-    BROWSER_SWITCH_SUCCEEDED("local-payment:start-payment:browser-presentation:succeeded"),
-    BROWSER_SWITCH_FAILED("local-payment:start-payment:browser-presentation:failed"),
```

Comment:
I'm fine removing these as they don't add much value in this flow, but we should maybe align the approach with this PR (#904) where I was just sending the browser failed event with the tokenize failed event. What do you think makes more sense?

Reply:
I feel that it might be redundant to send multiple events back to back for a certain scenario. I think this is where adding detail params would be able to capture the same data while minimizing the amount of events we're sending.

Should I align the PR to match the one you shared or create a story to ensure we're sending the browser switch detail for this error?

Reply:
Ok agreed let's keep this PR as is then and can update the PayPal analytics when the detail is added to the error in a future PR.